etc/adminhtml/system.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="web">
+            <group id="upward" translate="label" type="text" sortOrder="300"
+                showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Assuming we're only supporting values for default scope, hide for website and store.